code/network/multi_campaign.cpp

```diff
@@ -40,13 +40,15 @@
 #define MC_CODE_DONE									1			// campaign is "done"
 #define MC_CODE_DEBRIEF								2			// debrief info
 #define MC_CODE_START								3			// start campaign information
+#define MC_CODE_MISSION_NAMES						4			// mission names
 
 // packet code for ingame joining
 #define MC_JIP_INITIAL_PACKET						1			// initial data 
 #define MC_JIP_GE_STATUS							2			// goal event status for all missions
 #define MC_JIP_GOAL_NAMES							3			// goal names
 #define MC_JIP_EVENT_NAMES							4			// event names
-#define MC_JIP_END_DATA								5			// this is the end of the data
+#define MC_JIP_MISSION_NAMES						5			// mission names
```

Comment:
Can these be enum's or enum classes?

Reply:
Probably, but I didn't want to change it just to change it.